docs/conf.py

```diff
@@ -0,0 +1,110 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+# sys.path.insert(0,os.path.dirname(sys.path[0]))
+
+# -- Project information -----------------------------------------------------
+
+project = "WaterTAP-SETO"
+copyright = "2022-2023, NREL et al."
+author = "WaterTAP-SETO dev team"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.0dev0"
+# The short X.Y version
+version = "0.1.0dev0"
+# -- General configuration ---------------------------------------------------
+
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx_rtd_theme",
+    "sphinx.ext.napoleon",  # Google and NumPy-style docstrings
+    "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.coverage",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.ifconfig",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.githubpages",
+    "sphinx.ext.autosectionlabel",
+    "sphinx.ext.doctest",
+    "nbsphinx",  # Jupyter notebooks as docs
```

Comment:
```suggestion
    "sphinx_rtd_theme",
    "sphinx.ext.napoleon",  # Google and NumPy-style docstrings
    "sphinx.ext.autodoc",
    "sphinx.ext.intersphinx",
    "sphinx.ext.mathjax",
    "sphinx.ext.ifconfig",
    "sphinx.ext.viewcode",
    "sphinx.ext.autosectionlabel",
    "sphinx.ext.doctest",
```

Not sure all of these are needed; in general having more than you strictly need shouldn't outright cause issues (assuming you specify the corresponding dependencies as needed) but it makes sense to exclude the ones you know you don't need.